src/platform_impl/apple/appkit/app_state.rs

```diff
@@ -117,6 +117,12 @@ impl AppState {
         // to allow the package manifest to define behavior via LSUIElement.
         if self.activation_policy.is_some() {
             app.setActivationPolicy(self.activation_policy.unwrap());
+        } else {
+            let is_bundle =
+                unsafe { NSRunningApplication::currentApplication().bundleIdentifier().is_some() };
+            if !is_bundle {
```

Comment:
Please add a comment here, perhaps linking to https://github.com/rust-windowing/winit/issues/3958